dod-advana/gamechanger-web
# Task/UOT-108861 - Graph View Fixes

## Description


- Close the node menu after clicking the reset graph button
- Zoom and pan the graph to fit the nodes after reset
- Fixes issues with some details pages crashing

## Related Issue/Ticket




https://jira.di2e.net/browse/UOT-108861

## Testing instructions



1. Open a document graph
2. Alter graph (move nodes around, dismiss, show reference nodes, etc.)
3. Click reset graph button. Graph should zoom and pan to fit the nodes
4. Click on a node to get the button popups
5. Click reset graph button. Menu should close
